### PR TITLE
Enabled the container based Travis CI architecture

### PR DESCRIPTION
**Pull request by Mailaender**
After some back https://github.com/OpenRA/OpenRA/pull/7311 and forth https://github.com/OpenRA/OpenRA/pull/8187 we finally need to upgrade. I had to revert https://github.com/OpenRA/OpenRA/pull/7510 because that is not available nor install-able in the new @Travis-CI environment. http://docs.travis-ci.com/user/migrating-from-legacy

### CODE REVIEW

File: packaging/windows/buildpackage.sh [hunk 1]
```diff
@@ -8,11 +8,6 @@ OUTPUTDIR="$4"
 if [ -x /usr/bin/makensis ]; then
     pushd "$SRCDIR" >/dev/null
     popd >/dev/null
```
Comment (obrakmann) [inline packaging/windows/buildpackage.sh]: These two lines can go away. I probably messed up there when copying&pasting.

**Comment by obrakmann** [comment]
Local builds still work fine, :+1:

File: packaging/windows/buildpackage.sh (hunk 1 above)
Comment (Mailaender) [inline packaging/windows/buildpackage.sh]: Agreed. Removed.

**Comment by pchote** [comment]
Have we tested that the nsis packages generated by this still install and uninstall properly?

**Comment by reaperrr** [comment]
> Have we tested that the nsis packages generated by this still install and uninstall properly?

They do, under Win7x64 at least.

**Comment by reaperrr** [comment]
:+1:

**Pull request merged**